src/main/java/com/zendesk/maxwell/producer/DataOutput.java

```diff
@@ -0,0 +1,55 @@
+package com.zendesk.maxwell.producer;
+
+/**
+ * Enum to indicate if the data parameter should be the unique parameter in the output
+ *
+ * example:
+ *
+ * data output = false
+ *
+ *   maxwell: {
+ *	 	"database": "test",
+ *	 	"table": "maxwell",
+ *	 	"type": "insert",
+ *	 	"ts": 1449786310,
+ *	 	"xid": 940752,
+ *	 	"commit": true,
+ *	 	"data": { "id":1, "daemon": "Stanislaw Lem" }
+ *	 }
+ *
+ * data output = true
+ *
+ *   maxwell: {
+ *	 	"data": { "id":1, "daemon": "Stanislaw Lem" }
+ *	 }
+ *
+ * data output = content
+ *
+ *   maxwell: {
```

Comment:
I like the way you have implemented this. It is nice an flexible however for data output - content we literally want the content i.e without the "maxwell" key. So just
 {
 "id":1,
 "daemon": "Stanislaw Lem"
}